packages/platforms/celestica/x86-64/x86-64-cel-redstone-xp/onlp/builds/src/module/src/sys_eeprom.h

```diff
@@ -0,0 +1,186 @@
+#ifndef SYS_EEPROM_HH
```

Comment:
Why didn't you use the existing ONIE infrastructure for all of this?
https://github.com/opencomputeproject/OpenNetworkLinux/blob/master/packages/base/any/onlp/src/onlplib/module/inc/onlplib/onie.h